### DISCUSSION TITLE
How to configure rocket chip?

### DISCUSSION THREAD

**Discussion by pa1ly** [chipsalliance/rocket-chip#3213] [Q&A] [answered]
There are no configuration options available in src/scala/system/Config.scala as mentioned in the readme file.

**Comment by rgaehfgoiarewhg** [comment]
welcome to hardware design.

**Comment by seldridge** [comment, marked as answer]
There are top-level "system" configurations that define options and pull-in "sub-system" configurations. The following two files are a good place to start:
  - [src/main/scala/system/Configs.scala](https://github.com/freechipsproject/rocket-chip/blob/master/src/main/scala/system/Configs.scala)
  - [src/main/scala/subsystem/Configs.scala](https://github.com/freechipsproject/rocket-chip/blob/master/src/main/scala/subsystem/Configs.scala)
